spaceR/spaceR/Models/Launchpads.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Launchpads.swift
+//  spaceR
+//
+//  Created by Kieran Woodrow on 2022/03/16.
+//
+
+import Foundation
+
+struct Launchpads: Codable {
+    
+    let launchID: String?
+    let lauchpadName: String?
+    let launchpadStatus: String?
+    let location: String?
+    let region: String?
+    let timezone: String?
+    let latitude: Double?
+    let longitude: Double?
+    let launchAttempts: Int?
+    let launchSuccesses: Int?
+    let rocketsLaunched: [UUID?]
```

Comment:
Hmm this seems a little bit strange, what type are you expecting here? 😄

Reply:
This in an example return of the json

 "longitude": -120.5931438,
        "launch_attempts": 0,
        "launch_successes": 0,
        "rockets": [
            "5e9d0d95eda69955f709d1eb"
        ],

This is the schema declared in the backend
  "latitude": {
    "type": "Number",
    "default": null
  },
  "longitude": {
    "type": "Number",
    "default": null
  },
  "launch_attempts": {
    "type": "Number",
    "default": 0
  },
  "launch_successes": {
    "type": "Number",
    "default": 0
  },
  "rockets": [
    "UUID"
  ],

Reply:
Hi @kieranwoodrow So that value is still just an array of `String` UUID just means that it will be a unique identifier, but you are not actually expecting a type of UUID.

Reply:
Judging by this response you might also need to take another look at your model because  most of these need to decode into an object and not just an `Int` for example longitude will not decode into an `Double`

Reply:
Okay, I changed it to String and not UUID, but i just tried to make an api call now and it returned and  mapped everything alright and i can access everything i need😅

Reply:
Remember to push your code 😄